engines/plos_authors/app/controllers/plos_authors/plos_authors_controller.rb

```diff
@@ -8,7 +8,7 @@ class PlosAuthorsController < ApplicationController
     def create
       plos_author.save!
       ActivityFeed.create(
-        feed_name: 'workflow',
+        feed_name: 'manuscript',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.